src/uproot/models/RNTuple.py

```diff
@@ -563,9 +589,14 @@ def arrays(
             if "column" in key and "union" not in key:
                 key_nr = int(key.split("-")[1])
                 dtype_byte = self.column_records[key_nr].type
+
+                # Check if column stores offset values for jagged arrays (applies to cardinality cols too):
+                is_offset_col = dtype_byte == 14
```

Comment:
Instead of hardcoding 14, it would be more readable to use `uproot.const.rntuple_col_type_to_num_dict["splitindex64"]`. I imagine this also applies to `splitindex32`.

Reply:
Good idea, I wanted to do that, but did not find which variables to use.
I will edit PR.